### PR TITLE
Dev

### LINKED ISSUE #15: Segregate CLI from game controller & set the GUI to be interchangeable

### PR DESCRIPTION
**Pull request by benlapETS**
Completed the whole application:

- Corrected issues from PR #16
- Made UIs interchangeables
- Adapted the tests

### CODE REVIEW

File: hanggame/game.py [hunk 1]
```diff
@@ -16,113 +16,63 @@
...
         randomly choose an item from a list or basically a sequence.
-
-*constant*
-    **EMPTY_STR***
-        An empty string constants
-    **YES_NO_LIST**
-        lists of yes/no options that the player may use to answer the play again
-        question.
 """
-import logging
```
Comment (labesoft) [inline hanggame/game.py:27]: missing copyright

File: hanggame/hangman.py [hunk 2]
```diff
@@ -26,38 +28,46 @@
...
+]
+PART_ARM_LEFT = "{left}|{middle}/|"
+PART_ARMS_HANGED = "{left}|{middle}/|\\"
+PART_BODY = "{left}|{middle} |"
+PART_HEAD_HANGED = "{left}|{middle} O"
+PART_HEAD_SAVED = "{left}|{middle}\\O/"
+PART_LEG_LEFT = "{left}|{middle}/"
+PART_LEGS_HANGED = "{left}|{middle}/ \\"
+PART_LEGS_SAVED = "{lfoot}__|__{rfoot}/ \\"
+
```
Comment (labesoft) [inline hanggame/hangman.py:53]: not alpha ordered

**Review by ghost — APPROVED** [review]
Good job, left some minor comments.

File: hanggame/__main__.py [hunk 3]
```diff
@@ -51,5 +53,10 @@
...
     # Game
-    game = HangGame(level=list(GameLevel)[args.level])
-    game.run()
+    level = list(GameLevel)[args.level]
+    word = Word()
+    word.choose()
+    if not args.gui:
+        console.main(level, word)
+    else:
+        window.main(level, word)
```
[the remark this answers is not in the document]
Comment (benlapETS) [reply hanggame/__main__.py]: Done

File: hanggame/game.py (hunk 1 above)
Comment (benlapETS) [reply hanggame/game.py:27]: Done... in all modules

File: hanggame/hangman.py (hunk 2 above)
Comment (benlapETS) [reply hanggame/hangman.py]: Done

File: README.md [hunk 4]
```diff
@@ -8,9 +8,9 @@ level up our programming skills. For more information you may visit the [data-fl
 website](https://data-flair.training/blogs)
 
 ## Projects Included 
-* [Alarm Clock with GUI](alarm/README.md)
-* [Hangman Game in Python](hanggame/README.md)
```
[the remark this answers is not in the document]
Comment (benlapETS) [reply README.md:12]: Done

**Pull request merged**